### DISCUSSION TITLE
DotNetCore.Natasha.CSharp 包怎么没了

### DISCUSSION THREAD

**Discussion by myesn** [dotnetcore/Natasha#373] [Q&A] [answered]
hi 各位，我有一个老项目使用了 `DotNetCore.Natasha.CSharp` 包，版本是 `5.2.2.1`，今天在 nuget 上搜索发现找不到了，请问新的包名是什么呢？谢谢告知

**Comment by NMSAzulX** [comment, marked as answer]
`DotNetCore.Natasha.CSharp.Compiler.Domain` 和 `DotNetCore.Natasha.CSharp.Compiler`, [文档](https://natasha.dotnetcore.xyz/zh-Hans/docs/compile/)
